src/ResourceManager/AnalysisServices/Commands.AnalysisServices/Models/AzureAnalysisServicesServer.cs

```diff
@@ -79,6 +80,10 @@ internal static AzureAnalysisServicesServer FromAnalysisServicesServer(AnalysisS
                 config = new PsAzureAnalysisServicesFirewallConfig(enablePowerBIService, rules);
             }
 
+            var rgStart = server.Id.IndexOf("resourceGroups/", StringComparison.InvariantCultureIgnoreCase) + ("resourceGroups/".Length);
```

Comment:
@taiwu nit: please use the [`ResourceIdentifier`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Common/Commands.ResourceManager.Common/Utilities/Models/ResourceIdentifier.cs) class to parse the resource id and grab the resource group name

Reply:
Updated, include the property check in our tests.